docs/changelog.md

```diff
@@ -1,5 +1,30 @@
 # NEURON 8.2
 
+## 8.2.2
+_Release Date_ : 15-12-2022
+
+### What's New
+
+- Python 3.11 support
+- Re-enabled MUSIC support in NEURON (#1896)
+- CVode.poolshrink(1) deletes unused mechanism pools. (#2033)
+
+### Bug Fixes
+
+- Bugfix for rxd reactions involving rxd parameters. #1933
```

Comment:
There was a * check for NULL. (already fixed in master 962533e7) bugfix
commit b246509cd2
in bgpdma.cpp
But perhaps that was too trivial to mention. I just remember it caused a bit of consternation